src/p4est_communication.c

```diff
@@ -595,6 +595,21 @@ p4est_comm_is_empty_gfq (const p4est_gloidx_t *gfq, int num_procs, int p)
   return gfq[p] == gfq[p + 1];
 }
 
+int
+p4est_comm_is_empty_gfx (const p4est_gloidx_t *gfq, const p4est_quadrant_t *gfp,
+                         int num_procs, int p)
+{
+  P4EST_ASSERT (0 <= p && p < num_procs);
+  P4EST_ASSERT (gfp != NULL);
+
+  if (gfq) {
```

Comment:
Please check pointers explicitly with == NULL or  != NULL.

Reply:
Good point, I've changed this to `gfq != NULL` now.